packages/Uploader/src/components/File/File.styles.js

```diff
@@ -0,0 +1,92 @@
+import stylers from "@paprika/stylers";
+import tokens from "@paprika/tokens";
+import styled, { css } from "styled-components";
+import statuses from "../../statuses";
+
+export const FileWrapper = styled.div`
+  align-items: center;
+  border-bottom: 1px solid ${tokens.color.blackLighten40};
+  display: flex;
+  padding: ${stylers.spacer(2)};
+`;
+
+export const Left = styled.div`
+  flex: 1;
+`;
+
+export const Right = styled.div`
+  display: flex;
+  flex: 0 0 ${stylers.spacer(4)};
```

Comment:
```suggestion
  flex-shrink: 0;
  width: ${stylers.spacer(4)};
```

Reply:
☝️ I almost never use `flex-basis`, and am not even sure what the difference would be in this case vs `width`.  Do you?

Reply:
@mikrotron Nah I think width and flex-basis do the same thing when flex-direction is `row`.
But I think it makes sense to use `flex-basis` instead of `width` when using flexbox.

Reply:
> The first thing that comes to mind when reading your question is that flex-basis doesn't > always apply to width.
>
> When flex-direction is row, flex-basis controls width.
> 
> But when flex-direction is column, flex-basis controls height.

Reply:
sound a complex question to answer https://stackoverflow.com/a/34355447/196038 :P